source/beamline/VESPERS/VESPERS13ElementGeDetector.h

```diff
@@ -0,0 +1,71 @@
+#ifndef VESPERS13ELEMENTGEDETECTOR_H
+#define VESPERS13ELEMENTGEDETECTOR_H
+
+#include "beamline/AMXRFDetector.h"
+
```

Comment:
In general there seems to be a lot of similarities between this an the IDEAS version. Is there potential here for a common ancestor class?